### ISSUE TITLE
Ignore tables?

### ISSUE THREAD

**Issue by gburnett** [acarl/pg-restify#14]
Is it possible to specify any tables that should be ignored?

**Comment by acarl** [comment]
The simplest way to achieve this would be to use addPreHookForResource. 
If you look at the documentation on hooks you can see an example of one where an alternative response is returned. In your case I would imagine you would want to return a 404 response.

```js
hooks.addPreHookForResource('get', 'table-to-ignore', function(req, res, dbClient, next) {
 
  // You can choose to execute the callback with an error to halt further execution.
  return next(new restify.NotFoundError());
 
});
```

The downside to this is that you have to add one of these for each operation and table, but you could add some code to simplify that.

It wouldn't be too hard to add a configuration option to the code as well if that would be helpful for you. I just wanted to offer you something else that you could use immediately.

**Comment by gburnett** [comment]
I've just realised that my question was a bit rubbish.

I actually want to ignore the tables when I initialize.  I have some tables with composite primary keys so initialization fails.

I assume that it would be best to ignore these tables as I can't specify an `id` column.

**Comment by acarl** [comment]
I checked in a change to master which should get you what you are looking for. 
If that works for you I could cut a new release with the change. 
Really the whole codebase could use a refresh, but I haven't been able to get back to it for a while.

**Comment by gburnett** [comment]
Thanks, I'll take a look later today.

**Comment by gburnett** [comment]
@acarl Looks like it would work fine for one table.  With multiple tables it fails due to the sql syntax.

You should do something to add a comma between parameters around [here](https://github.com/acarl/pg-restify/commit/f58da6b64c99e5aa2c14687f0342352b51c22cf3#diff-6d186b954a58d5bb740f73d84fe39073R493)

You might also want to add a second table to the test [here](https://github.com/acarl/pg-restify/commit/f58da6b64c99e5aa2c14687f0342352b51c22cf3#diff-a566bf7e955b470f73fb4af984b65f7eR132)

**Comment by acarl** [comment]
Thanks for your help with the pull request. 
Some day I'll get around to updating this code base. It really uses a lot of out-of-date JS patterns currently.
I also updated some of the main dependencies and verified the tests still pass. I published a new release as v0.6.0. Let me know if you see any other issues.
